README.md

```diff
@@ -97,11 +97,16 @@ Log in as the buildbot user, and do the following:
     virtualenv --no-site-packages buildbot-env
     source buildbot-env/bin/activate
     echo "export PATH=/home/buildbot/buildbot-ros/scripts:${PATH}" >> buildbot-env/bin/activate
-    easy_install buildbot
+    easy_install buildbot=0.8.12 requests
     pip install rosdistro
     git clone git@github.com:mikeferguson/buildbot-ros.git
     buildbot create-master buildbot-ros
 
+If using the Pull Request builder, you will also need to:
+
+    sudo apt-get install libffi-dev libssl-dev
+    easy_install txgithub
```

Comment:
If you didn't run into any problems setting up then I don't think there need to be any more doc changes, though.